### PR TITLE
Fix webpack using valid image file for default icon path (#4849)

### PR DESCRIPTION
**Pull request by jefbarn**

### CODE REVIEW

File: src/layer/marker/Icon.Default.js [hunk 1]
```diff
@@ -42,8 +42,9 @@ L.Icon.Default = L.Icon.extend({
 		           L.DomUtil.getStyle(el, 'backgroundImage');	// IE8
 
 		document.body.removeChild(el);
+		console.log('marker path', path);
```
Comment (IvanSanchez) [inline src/layer/marker/Icon.Default.js]: No `console.log()`s, plz

**Comment by IvanSanchez** [comment]
Seems to work in FFX and phantom unit tests. Will make a couple tests in a couple more browsers tomorrow.

Looks good, except for the `console.log()`.

**Comment by jefbarn** [comment]
Oops, forgot to pull that log message out, I'll fix it up.

**Comment by jefbarn** [comment]
Grrr, I tried to fixup the commit messages, but just ended up making it worse.  I hate git sometimes.

**Comment by mourner** [comment]
Let's try it out!

**Comment by IvanSanchez** [comment]
Works for me. No problems on FFX, chrome nor ie11, tests pass.

**Comment by lmcarreiro** [comment]
This pull request broke the case with webpack:

image

It's common to use `[name]-[hash].[ext]` with file-loader in webpack to avoid browser cache issues, the regex should be more flexible.

**Pull request merged**